### ISSUE TITLE
Integrate Nao in NVDA community translation framework

### ISSUE THREAD

**Issue by CyrilleB79** [sharkboyto/nao#23]
I have already discussed it by e-mail. But I open this issue here in order to have it tracked.

I think that Nao should be integrated in the NVDA community translation framework. This would help translators to do their job more easily for the following reasons:
1. Provides a standard way to indicate to translators when the translations need to be done.
2. No need to merge manually the .pot in .po
3. When updating the documentation, no need to remember which version of the doc was last translated and to check the diff with this version.
This 3rd point is a real benefit to have the documentation translated and up-to-date. On the contrary, in manual translation process, updating the documentation is a burdon when it becomes bigger.

The automatic system has also the following benefits for the add-on maintainner: 
1. Be able to merge all translations in one single merge.
2. Notify translators of new translation content only when you want, e.g. only when a new version of the add-on is released.

Note: a pre-requisite to the inclusion in the international community translation system is to have the addd-on integrated on the international community website.
If needed, I can help for both these tasks, in English, Italian or French.

**Comment by nvdaes** [comment]
Hello: I want to support Cyrille's request. I would like to see this add-on on the [add-ons website available from NVDA add-ons manager](https://addons.nvda-project.org), and registered in the translation system.
